kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/ProcessResource.java

```diff
@@ -327,6 +327,8 @@ public Response startProcessWithCorrelationKeyFromNodeIds(@javax.ws.rs.core.Cont
                             MessageFormat.format(PROCESS_DEFINITION_NOT_FOUND, processId, containerId), v);
         } catch (SecurityException e) {
             return forbidden(errorMessage(e, e.getMessage()), v, conversationIdHeader);
+        } catch (IllegalArgumentException e) {
+            return badRequest(errorMessage(e, e.getMessage()), v, conversationIdHeader);
         } catch (Exception e) {
             logger.error("Unexpected error during processing {}", e.getMessage(), e);
```

Comment:
look up the excepton  and react in here not at jbpm level.

Reply:
@elguardian added exception handling code at rest layer. Please take a look.